### DISCUSSION TITLE
请问skynet.lua的报错是什么意思。。。

### DISCUSSION THREAD

**Discussion by joeplayer126** [cloudwu/skynet#1618] [Q&A]
报错如下：
./common/lualib/skynet.lua:596: Invalid req package type 10
./common/lualib/skynet.lua:596: Invalid req package type 110
./common/lualib/skynet.lua:596: Invalid req package type 7
./common/lualib/skynet.lua:596: Invalid req package type 106
./common/lualib/skynet.lua:596: Invalid cluster message (size=0)
./common/lualib/skynet.lua:596: Invalid cluster message (size=3)

![image](https://user-images.githubusercontent.com/49432395/177093438-37d000d4-db44-46f1-bb52-c7a8415e9386.png)

skynet版本1.2

**Comment by terry8210** [comment]
升级版本再看看

**Comment by cloudwu** [comment]
应该是内网没有设置防火墙，接收到了一些未定义的包。
